roles/docker_storage_driver/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Start docker.service
+  service:
+    name: docker
```

Comment:
``openshift.docker.service_name`` might be used here if ``container-engine`` is also supported.

Reply:
Yeah, about that.  What would be the equivalent to `docker info` when using something other than Docker?

In my playbook that uses this role, I run `docker info` at the beginning and extract the "Storage Driver" value.  If it's already `"overlay2"` I skip the whole thing so the playbook is idempotent.  I'd like to preserve that feature when using `container-engine` or whatever else.

Reply:
``docker info`` would be the same. ``docker`` client is installed and will use ``container-engine`` service.